packages/ocaml/ocaml.4.12.0/opam

```diff
@@ -6,9 +6,10 @@ and polls it to initialise specific variables like `ocaml:native-dynlink`"""
 maintainer: "platform@lists.ocaml.org"
 depends: [
   "ocaml-config"
-  "ocaml-base-compiler" {= "4.12.0"} |
-  "ocaml-variants" {>= "4.12.0" & < "4.12.1~"} |
-  "ocaml-system" {>= "4.12.0" & < "4.12.1~"}
+  "ocaml-base-compiler" {>= "4.12.0~" & < "4.12.1~"} |
+  "ocaml-variants" {>= "4.12.0~" & < "4.12.1~"} |
+  "ocaml-system" {>= "4.12.0" & < "4.12.1~"} |
+  "metaocaml" {>= "4.12.0~" & < "4.12.1~"}
```

Comment:
I think the `metaocaml` line go in a separate PR as and when that's moved

Reply:
Right.
```suggestion
```